packages/animations/test/open_container_test.dart

```diff
@@ -1694,6 +1694,50 @@ void main() {
     expect(tester.getSize(find.text('Opened')),
         equals(tester.getSize(find.byKey(appKey))));
   });
+
+  testWidgets(
+    'Verify routeSettings passed to Navigator',
+    (WidgetTester tester) async {
+      final OpenContainerNavigatorObserver observer =
+          OpenContainerNavigatorObserver();
+
+      const RouteSettings routeSettings = RouteSettings(
+        name: 'route-name',
+        arguments: 'arguments',
+      );
+
+      final Widget openContainer = OpenContainer(
+        routeSettings: routeSettings,
+        closedBuilder: (BuildContext context, VoidCallback action) {
+          return GestureDetector(
+            onTap: action,
+            child: const Text('Closed'),
+          );
+        },
+        openBuilder: (BuildContext context, VoidCallback action) {
+          return GestureDetector(
+            onTap: action,
+            child: const Text('Open'),
+          );
+        },
+      );
+
+      await tester.pumpWidget(
+        MaterialApp(
+          home: openContainer,
+          navigatorObservers: <NavigatorObserver>[observer],
+        ),
+      );
+
+      // Open the container
+      await tester.tap(find.text('Closed'));
+      await tester.pumpAndSettle();
+
+      // Expect the last route pushed to the navigator to contain RouteSettings
+      // equal to the RouteSettings passed to the OpenContainer
+      expect(observer.lastRoutePushed.settings, routeSettings);
```

Comment:
Instead of defining an Observer, the test could just use `ModalRoute.of(tester.element(find.text('Open')))` to get the ModalRoute and check that `ModalRoute.settings` is set correctly to simplify things a little.

Reply:
@gmoothart Interesting, I've just made the suggested change!

Reply:
Mentioned the wrong person, sorry!
cc @goderbauer